app/models/fact_value.rb

```diff
@@ -15,6 +15,7 @@ class FactValue < ActiveRecord::Base
   scoped_search :relation => :host,      :on => :name, :complete_value => true, :rename => :host, :ext_method => :search_by_host_or_hostgroup, :only_explicit => true
   scoped_search :relation => :hostgroup, :on => :name, :complete_value => true, :rename => :"host.hostgroup", :ext_method => :search_by_host_or_hostgroup, :only_explicit => true
   scoped_search :relation => :fact_name, :on => :short_name, :complete_value => true, :aliases => ["fact_short_name"]
+  scoped_search :relation => :fact_name, :on => :type, :complete_value => false, :only_explicit => true, :aliases => ["origin"]
```

Comment:
Why `complete_value` is false? There are only 3 or 4 options in a typical installation, which makes it a good candidate for autocompletion IMO. Same for `only_explicit`, why?